packages/augur-sdk/src/constants.ts

```diff
@@ -84,6 +84,15 @@ export enum ControlMessageType {
   WebsocketError = "WebsocketError",
 }
 
+// Valid market liquidity spreads
+export enum MaxLiquiditySpread {
```

Comment:
augur-ui in `augur/packages/augur-ui/src/modules/common/constants.ts`

```
export const MAX_SPREAD_ALL_SPREADS = 'all';
export const MAX_SPREAD_20_PERCENT = '20';
export const MAX_SPREAD_15_PERCENT = '15';
export const MAX_SPREAD_10_PERCENT = '10';
```
these can be mapped to the reference the new ones.

example:

```
export const MAX_SPREAD_10_PERCENT  = Getters.Markets.MaxLiquiditySpread.TenPercent

```

Reply:
Ok, will update this later tonight when I get a chance.